src/main/scala/beam/agentsim/agents/choice/mode/ModeChoiceTransitIfAvailable.scala

```diff
@@ -5,18 +5,20 @@ import beam.router.Modes
 import beam.router.RoutingModel.EmbodiedBeamTrip
 import beam.sim.BeamServices
 
+import scala.collection.mutable.ArrayBuffer
+
 /**
   * BEAM
   */
 class ModeChoiceTransitIfAvailable(val beamServices: BeamServices) extends ModeChoiceCalculator {
 
   override def clone(): ModeChoiceCalculator = new ModeChoiceTransitIfAvailable(beamServices)
 
-  override def apply(alternatives: Seq[EmbodiedBeamTrip]) = {
-    var containsTransitAlt: Vector[Int] = Vector[Int]()
+  override def apply(alternatives: Seq[EmbodiedBeamTrip]): Option[EmbodiedBeamTrip] = {
```

Comment:
Same as other alt comments. I'll stop here and this should just be repeated for any future instances like this.